tests/config/test_configtypes.py

```diff
@@ -1378,8 +1378,7 @@ def test_validate_exists_not_abs(self, os_path):
         os_path.expanduser.side_effect = lambda x: x
         os_path.isfile.return_value = True
         os_path.isabs.return_value = False
-        with pytest.raises(configexc.ValidationError):
-            self.t.validate('foobar')
+        self.t.validate('foobar')
```

Comment:
Changing this test makes sense, as relative paths work now. Could you please also rename the test to `..._rel` instead `..._not_abs` (and the comment to say `relative` instead of `not absolute`) as this is the "normal case" now?

Also, I think below you could add `os_path.isfile.assert_called_once_with(...)` and the same for `os_path.isabs` to make sure the paths get expanded correctly with `standarddir.config()`.